publisher-sdk/src/main/java/com/criteo/publisher/model/User.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.model
+
+import com.criteo.publisher.annotation.OpenForTesting
+
+@OpenForTesting
+data class User(
+    val deviceId: String?,
+    val deviceIdType: String,
```

Comment:
nit: this is not expected to be variable, but a constant (`android`), making it variable could add some interrogation and increase the cognitive load